configs/config.js

```diff
@@ -4,27 +4,27 @@ import {
   homepage,
   license,
   name,
-  version
+  version,
 } from '../package.json'
 
 const babelSetup = {
   babelrc: false,
-  presets: [['es2015', { modules: false }]],
-  plugins: ["transform-class-properties"],
-  exclude: 'node_modules/**'
+  presets: [['@babel/preset-env', { modules: false }]],
+  plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
r u sure u dont want to use `loose` mode? would make ur code more performant and would result in less bytes

Reply:
Yep, yep—that is not intentional. 🙏